lib/my_inject.rb

```diff
@@ -0,0 +1,24 @@
+class Array
+  def my_inject accum=first, sym="", &bloc
+    #accum = self.first if accum == ""
+    n = count
+    x = 0
+    if sym != ""
+      while x <= n - 1
+        y = self[x]
+        accum = eval("#{accum} #{sym} #{y}")
+        x += 1
+      end
+    else
+      while x <= n - 1
+        accum = bloc.call(accum, self[x])
+        x += 1
+      end
+    end
+      puts accum
```

Comment:
Inconsistent indentation detected.<br>Do not write to stdout. Use Rails' logger if you want to log.